sdk/storage/azure-storage-file-share/tests/test_directory.py

```diff
@@ -427,6 +427,42 @@ def test_list_subdirectories_and_files(self, storage_account_name, storage_accou
         self.assertEqual(list_dir[5]['name'], 'file3')
         self.assertEqual(list_dir[5]['is_directory'], False)
 
+    @FileSharePreparer()
+    def test_list_subdirectories_and_files_encoded(self, storage_account_name, storage_account_key):
+        self._setup(storage_account_name, storage_account_key)
+        share_client = self.fsc.get_share_client(self.share_name)
+        directory = share_client.create_directory('directory\uFFFE')
```

Comment:
So just to confirm, like .NET, we don't expose this top-level directory anywhere? The one that is the top-level `DirectoryPath` in the XML response?

Reply:
If I am understanding your question correctly, yes it would seem that we don't expose this top-level directory (in the return from `get_directory_properties()` API call and in the return from `list_directories_and_files()` API call)

Reply:
Cool, just checking in case we missed it.